hubtools/src/lib.rs

```diff
@@ -652,23 +716,67 @@ impl RawHubrisArchive {
 
         let board = manifest
             .as_table()
-            .ok_or(Error::BadTomlType)?
+            .ok_or(Error::BadTomlTypeBoard)?
             .get("board")
-            .ok_or(Error::BadTomlType)?
+            .ok_or(Error::BadTomlTypeBoard)?
             .as_str()
-            .ok_or(Error::BadTomlType)?
+            .ok_or(Error::BadTomlTypeBoard)?
             .to_owned();
         let mut name = manifest
             .as_table()
-            .ok_or(Error::BadTomlType)?
+            .ok_or(Error::BadTomlTypeBoard)?
```

Comment:
Wrong error type here, should this be `BadTomlTypeName`?

Reply:
I had changed these error types as part of debugging. I'm reverting them.